examples/tutorial_examples/add_custom_bundle.py

```diff
@@ -0,0 +1,233 @@
+"""
+=================================================
+Adding new bundles into pyAFQ (SLF 1/2/3 Example)
+=================================================
+
+pyAFQ is designed to be customizable and extensible. This example shows how you
+can customize it to define a new bundle based on a definition of waypoint and
+endpoint ROIs of your design.
+
+In this case, we add sub-bundles of the superior longitudinal fasciculus,
+based on work by Rami et al [1]_.
+
+We start by importing some of the components that we need for this example and
+fixing the random seed for reproducibility
+"""
+
+import os.path as op
+import plotly
+import numpy as np
+import shutil
+
+from AFQ.api.group import GroupAFQ
+import AFQ.api.bundle_dict as abd
+import AFQ.data.fetch as afd
+from AFQ.definitions.image import ImageFile, RoiImage
+import wget
+import os
+np.random.seed(1234)
+
+
+#############################################################################
+# Get dMRI data
+# ---------------
+# We will analyze eight subject from the Healthy Brain Network Processed Open
+# Diffusion Derivatives dataset (HBN-POD2) [2]_, [3]_. We'll use a fetcher to
+# get preprocessed dMRI data for eight of the >2,000 subjects in that study. The
+# data gets organized into a BIDS-compatible format in the `~/AFQ_data/HBN`
+# folder. These 12 subjects have very high quality data.
+# The fether returns this directory as study_dir:
+
+_, study_dir = afd.fetch_hbn_preproc(['NDARKP893TWU',
+'NDAREP505XAD',
+'NDARKT540ZW0',
+'NDARAG340ERT',
+'NDAREM757NBG',
+'NDARLL894HC3',
+'NDARFY525TL2',
+'NDARKV461KGZ',
+'NDARUC851WHU',
+'NDARMJ333WJM',
+'NDARJG687YYX',
+'NDARJA157YB3',
+])
+
+
+roi_urls = ['https://github.com/yeatmanlab/AFQ/raw/c762ca4c393f2105d4f444c44d9e4b4702f0a646/SLF123/ROIs/MFgL.nii.gz',
```

Comment:
Maybe we should explain what's going on here?

Reply:
Emphasizing perhaps that these ROIs can be files that the user created themselves, but that in this case they are simply files stored somewhere on the internet (which happens to be another software library with a very similar name - this could get confusing for someone who doesn't know all the history...)

Reply:
@arokem I just added this. Please edit as you see fit and I'll merge